### DISCUSSION TITLE
Rogue's den plugin disappeared from plugin hub

### DISCUSSION THREAD

**Discussion by codestafa** [runelite/runelite#15737] [Q&A] [answered]
Hello, I am trying to use the Rogues' Den plugin, but I am not able to search it up on the plugin hub. Not sure if it's temporarily or permanently gone. Thanks! 

![rogues](https://user-images.githubusercontent.com/78193284/192606749-79a3dc6d-03e3-495e-b20f-09d71f0839e6.PNG)

**Comment by Enriath** [comment, marked as answer]
Well, first off you're not searching the plugin hub there, you're searching the normal plugin list. Usually people are doing this the other way around. See [this](https://github.com/runelite/runelite/wiki/Information-about-the-Plugin-Hub#how-to-use-the-plugin-hub)

But more generally, we don't provide support for hub plugins. If they go missing, 99% of the time it's because they failed to build after a client update and they'd need to be fixed by their author. In this specific case, I can confirm that the plugin *is* still on the hub and hasn't broken, so follow the guide linked above.
